app/settings/surveys/targeted-surveys/targeted-edit.controller.js

```diff
@@ -142,7 +142,7 @@ function (
         _.each(items, function (item, index) {
             _.each($scope.survey.attributes, function (question) {
                 if (question.label === item.getAttribute('data')) {
-                    question.order = index + 1;
+                    question.priority = index + 1;
```

Comment:
Important to consider: priority is selected by users, when they sort attributes. We should never force a higher priority on items unless our UI let them set an equal priority in 2 items.

Reply:
(basically,we don't choose priority, users do)

Reply:
nvm, I just got it sorry